packages/rum-core/src/domain/rumEventsCollection/action/listenActionEvents.ts

```diff
@@ -59,29 +49,13 @@ export function listenActionEvents<ClickContext>({ onPointerDown, onStartEvent }
 
     addEventListener(
       window,
-      isExperimentalFeatureEnabled('dead_click_fixes') ? DOM_EVENT.POINTER_UP : DOM_EVENT.CLICK,
-      (startEvent: MouseEvent) => {
-        if (isValidMouseEvent(startEvent) && clickContext) {
+      DOM_EVENT.POINTER_UP,
+      (startEvent: PointerEvent) => {
+        if (isValidPointerEvent(startEvent) && clickContext) {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏for symmetry with pointer down
```suggestion
      (event: PointerEvent) => {
        if (isValidPointerEvent(event) && clickContext) {
```